evalutils/stats.py

```diff
@@ -41,6 +47,9 @@ def distance_transform_edt_float32(  # noqa: C901
         Used for output of distance array, must be of type float64.
     indices : ndarray, optional
```

Comment:
If you're adding type hints to the function signature then remove them from the docstring as they are now out of sync.

Reply:
Good point, I'll do that.

Reply:
Done.